runtime/vm/jvminit.c

```diff
@@ -3111,12 +3111,7 @@ checkDllInfo(void* dllLoadInfo, void* userDataTemp)
 		if ( (entry->loadFlags & FAILED_TO_UNLOAD) || ((entry->loadFlags & FAILED_TO_LOAD) && !(entry->loadFlags & FATAL_NO_DLL)) ) {
 			userData->success = JNI_OK;
 		}
-		/* free string buffer if necessary */
-		if ((entry->loadFlags & FREE_ERROR_STRING) && entry->fatalErrorStr) {
-			j9mem_free_memory(entry->fatalErrorStr);
-			entry->loadFlags &= ~FREE_ERROR_STRING;
-		}
-		entry->fatalErrorStr = NULL;			/* ensure that error is not reported more than once */
+		setErrorJ9dll(PORTLIB, entry, NULL, FALSE);			/* ensure that error is not reported more than once */
```

Comment:
Please use a single space before the comment (tabs should only be used for indentation).